Python/symtable.c

```diff
@@ -610,7 +610,7 @@ error_at_directive(PySTEntryObject *ste, PyObject *name)
         return 0; \
     } \
     Py_DECREF(o); \
-}
+} while(0)
```

Comment:
Can you please add one indentation level, so this line doesn't seem to be at the module level?

Reply:
I've also put the `do {` on a separate line since it could be a bit clearer with the additional indentation.